### ISSUE TITLE
uui-swatch: selection no longer works with space key

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco.UI#654]
### Which *exact* UI Library version are you using? For example: 1.0.1 - don't just write v1.

1.6.0-rc

### Bug summary

The selection of the swatch used to be possible using the `space` key, but doesn't seem to work anywork.
I noticed in with 1.6.0-rc in backoffice, but also seems to be the case here:
https://uui.umbraco.com/?path=/story/uui-color-swatches--overview

https://github.com/umbraco/Umbraco.UI/assets/2919859/491ebcef-6a80-44b3-990b-c67acd9f625d

### Steps to reproduce

Try make a selection of in `uui-swatch` or a swatch in `uui-swatches` using keyboard and hit `space` key to select.
It seems to uncheck right away.

**Comment by bjarnef** [comment]
Actually it seems to be an issue in both latest version and 1.6.0-rc

I can reproduce in both of these examples, with a single hit on space key.

https://github.com/umbraco/Umbraco.UI/pull/536#issuecomment-1802026991

However if holding space key down sometimes the swatch is selected, sometimes not and toggle fast between selection. I think the issue is it react on keypress and hit keyup, when releasing the key.

**Comment by bjarnef** [comment]
@iOvergaard @nielslyngsoe I think the root cause to this, may be how the selectable mixin works?

https://github.com/umbraco/Umbraco.UI/blob/04709c32cb81b058bdf7976b69533e5b793c49ab/packages/uui-base/lib/mixins/SelectableMixin.ts#L66

**Comment by bjarnef** [comment]
I can also reproduce on selectable cards:
https://uui.umbraco.com/?path=/story/uui-card-media--selectable

Holding down space key toggle toggle the selection - in my opinion first when key is released (`keyup` event).

**Comment by nielslyngsoe** [comment]
Hi @bjarnef 

I agree, you should be able to use spacebar to select.

Feel free to make PR, please make a new PR for each component.

Thanks
